lib/customer.rb

```diff
@@ -0,0 +1,23 @@
+require "./lib/order.rb"
+
+class Customer
+  attr_reader :order
+
+  def initialize(order = Order)
+    @order = order.new
+  end
+
+  def view_menu
+    @order.view_menu.each_with_index do |dish, index|
+      puts "#{index + 1}. #{dish[:dish]}: £#{'%.2f' % dish[:price]}"
```

Comment:
Normally I would suggest extracting this kind of presentation logic out into a class of it's own to keep it separate from your business logic, but given that this class is mostly just delegating responsibility down to an `Order` object I think you're probably ok here.